jsonschema-core.xml

```diff
@@ -135,13 +135,66 @@
                 data in several ways.
             </t>
             <t>
-                JSON Schema uses keywords to assert constraints on JSON instances or annotate those
-                instances with additional information.  Additional keywords are used to apply
-                assertions and annotations to more complex JSON data structures, or based on
-                some sort of condition.
+                A JSON Schema document is comprised of a set of keywords,
+                which are specified as properties in a JSON object.
+                The name of the keyword is used as the property name,
+                and any arguments to the keyword are provided as the value.
+                Keywords can be classified by their functions.
+                The most important classes are validation, annotation, and applicator keywords.
             </t>
             <t>
-                To facilitate re-use, keywords can be organized into vocabularies. A vocabulary
+                "Validation keywords" form the basis of JSON Schema.
```

Comment:
Is it true that we consider validation to be any more important than other uses? I was under the impression we very much wanted to avoid making it seem that way, but this language would seem to conflict.

Also I think you're changing established vocabulary (no pun intended) here, unless some discussion has happened already that I wasn't privy to -- the original term is "assertion", so unless there's good reason to do so it seems wise to avoid churning terminology, and to not do so alongside a PR which is more about making the doc read smoother.

Reply:
I think it's fair to say validation is the basis of JSON Schema—there's no part of JSON Schema that's not touched by validation in some manner. E.g. The difference between `allOf` and `anyOf` is only validation. There's no similar keyword where the difference is only annotation. (Maybe, hypothetically, there's a keyword that passes through only the validation result, but filters out annotation results.)

For "assertion" vs. "validation", you're right, but we seem to be using it inconsistently throughout the document. Maybe we should open a separate issue for that.

Reply:
I don't think others will agree with your first paragraph, especially so considering all the relevant assertion keyword are in a separate vocabulary even. But let's see.

Reply:
@awwright as you know, I won't have bandwidth to fully respond for another week, but a few quick things:

* While validation is foundational to many things, I agree with @Julian that we should not promote it as the leading case.  It is possible to use JSON Schema core, applicators, annotations, and locations (custom vocabularies aside from core) without any assertions.  I did this for a client to create a complex configuration system (the configurations could also be validated by JSON Schema, but they were not assertion-based themselves).
* "assertion" is the keyword classification, "validation" is the overall process which involves more than just assertions.
* In general, this is all directly overlapping and/or conflicting with my much-delayed (and, as you know, delayed again) keyword behavior presentation.  So I can't really give proper feedback until that's done, for the same reaons that I've declined to dribble out bits and pieces of the presentation.

Reply:
What would you say forms the "basis of JSON Schema"? Recursively enumerating the value space of JSON?

Reply:
I changed it to say "assertion keyword", as for what forms the "basis" of JSON Schema that seems to have picked up below.